specification/dataprotection/resource-manager/Microsoft.DataProtection/DataProtection/client.tsp

```diff
@@ -36,3 +36,44 @@ using Microsoft.DataProtection;
 );
 
 @@clientName(Microsoft.DataProtection, "DataProtectionMgmtClient", "python");
+
+@@clientName(BackupVaultOperationResults.getInResourceGroup,
```

Comment:
why this is only JS breaking?

Reply:
I don't know. Java was `get` in GA. And there is no change from this PR.

Reply:
Same as Go.